topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.md

```diff
@@ -0,0 +1,1753 @@
+
+<div class='Separate-Preprocessing-Steps' markdown='1'>
+><hands-on-title>Separate Preprocessing Steps</hands-on-title>
+>
+> 1. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `Filtered Dataset` (output of **Seurat Create** {% icon tool %})
+>    - *"Method used"*: `Normalize with 'NormalizeData'`
+>        - *"Method for normalization"*: `LogNormalize`
+>
+> 2. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of previous **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Identify highly variable genes with 'FindVariableFeatures'`
+>        - *"Method to select variable features"*: `vst`
+>        - *"Output list of most variable features"*: `Yes`
+>        - *"Number to show"*: `10`
+>
+> 3. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of previous **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Scale and regress with 'ScaleData'`
+>        - *"Regress out a variable"*: `No`
+>        - *"Features to scale"*: `All Features`
+>
+> We'll choose to scale `All Features` here, instead of just the highly variable genes.
+>
+> 4. Rename the output as `Preprocessed Data`
+>
+{: .hands_on}
+</div>
+
+<div class='SCTransform' markdown='1'>
+><hands-on-title>SCTransform</hands-on-title>
+>
+> 1. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `Filtered Dataset` (output of **Seurat Create** {% icon tool %})
+>    - *"Method used"*: `Complete all preprocessing with 'SCTransform'`
+>        - *"Genes to calculate residual features for"*: `all genes`
+>            - *"How to set variable features"*: `set number of variable features`
+>        - *"Output list of most variable features"*: `Yes`
+>        - *"Number to show"*: `10`
+>        - *"Variable(s) to regress out"*: `percent.mt`
+>
+> 2. Rename the output as `Preprocessed Data`
+>
+> If we inspect this dataset, we can see that a new assay called `SCT` has been created and is set as the default assay. Click on the {% icon galaxy-eye %}- of the following output to check this.
```

Comment:
If you line this text up with the text above it, it won't break the numbering in the box

```suggestion
>    If we inspect this dataset, we can see that a new assay called `SCT` has been created and is set as the default assay. Click on the {% icon galaxy-eye %}- of the following output to check this.
```